modules/serverless-functions-adding-annotations.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-annotations.adoc
```

Comment:
We don't need to include this anymore as part of OCP doc guidelines - it can be removed. I've actively been removing it from existing modules as well when I come across it 😉